hasura-test/backend/hasura/config.yaml

```diff
@@ -0,0 +1,6 @@
+version: 2
+endpoint: https://backend.lokaler.lnu.test
+metadata_directory: metadata
+actions:
+  kind: synchronous
+  handler_webhook_baseurl: http://localhost:3000
```

Comment:
Not sure what this does. It was in the tutorial, but not sure if it's the right address for our setup.